compile.c

```diff
@@ -14214,6 +14214,10 @@ ibf_load_setup_bytes(struct ibf_load *load, VALUE loader_obj, const char *bytes,
 static void
 ibf_load_setup(struct ibf_load *load, VALUE loader_obj, VALUE str)
 {
+    if (!RB_TYPE_P(str, T_STRING)) {
+        rb_raise(rb_eTypeError, "parameter must be a binary format String object");
+    }
```

Comment:
```suggestion
    StringValue(str);
```

It's more common to use `StringValue` when a method requires a string.

Reply:
Oh neat, thanks I'll keep that in mind in the future.

Reply:
Done, looks good local.